h2o-core/src/main/java/water/util/TwoDimTable.java

```diff
@@ -82,7 +82,7 @@ else if (colTypes.length != colDim)
         colTypes[c] = colTypes[c].toLowerCase();
         if (!(colTypes[c].equals("double") || colTypes[c].equals("float") || colTypes[c].equals("int") ||
             colTypes[c].equals("long") || colTypes[c].equals("string")))
-          throw new IllegalArgumentException("colTypes values must be one of \"double\", \"float\", \"int\", \"long\", or \"string\"");
+          throw new IllegalArgumentException(String.format("colTypes values must be one of \"double\", \"float\", \"int\", \"long\", or \"string\". Received \"%s\"", colTypes[c]));
```

Comment:
Can't we get the index too?

throw new IllegalArgumentException(String.format("colTypes values must be one of \"double\", \"float\", \"int\", \"long\", or \"string\". Received \"%s\" as colType #%d", colTypes[c], c));

Reply:
Good suggestion. Thanks. Will do.